Code/Framework/AzCore/AzCore/DOM/DomUtils.h

```diff
@@ -92,10 +92,10 @@ namespace AZ::Dom::Utils
     template<typename T>
     using DomValueWrapperType = typename DomValueWrapper<T>::Type;
 
-    static const AZ::Name TypeFieldName = AZ::Name::FromStringLiteral("$type");
-    static const AZ::Name PointerTypeName = AZ::Name::FromStringLiteral("pointer");
-    static const AZ::Name PointerValueFieldName = AZ::Name::FromStringLiteral("value");
-    static const AZ::Name PointerTypeFieldName = AZ::Name::FromStringLiteral("pointerType");
```

Comment:
FYI you could have inlined the variable instead using the inline keyword instead of `static` to make sure the linker would get one copy